marklogic-data-hub-central/ui/src/pages/Overview.test.tsx

```diff
@@ -164,7 +166,7 @@ describe("Overview component", () => {
     expect(documentationLinks.length === enabled.length); // All cards should have Documentation links
     documentationLinks.forEach((docLink, i) => {
       fireEvent.click(docLink);
-      expect(mockedWindowOpen).toBeCalledWith(overviewConfig.documentationLinks[enabled[i]], "_blank");
+      expect(mockedWindowOpen).toBeCalledWith(overviewConfig.documentationLinks.tileSpecificLink(5.3, enabled[i]), "_blank");
```

Comment:
Thanks @Sanjeevani19! In this line, I believe I have verified that when the environment data is set to 5.3, the mock window is opened with the "5.3" version number.